schema_based_on_diagram.sql

```diff
@@ -0,0 +1,39 @@
+create table patients(id serial primary key , name varchar(50) , date_of_birth date);
+
+create table invoices(
+    id serial primary key , 
+    total_amount decimal , 
+    generated_at timestamp ,
+    payed_at timestamp,
+    medical_history_id int,
+    constraint fk 
+    foreign key medical_history_id 
+    references medical_histories(id) 
+    on delete cascade
+    )
+
+create table medical_histories(
```

Comment:
- [ ] It would be better if we create a join table between the medical_histories and treatments. For example;


```sql
 CREATE TABLE medical_histories_has_treatments (
    medical_history_id int refrences medical_histories(id),
    treatment_id int refrences treatments(id),
    );
```